app/src/pages/inside/projectSettingsPageContainer/content/patternAnalysis/modals/editPatternModal/editPatternModal.jsx

```diff
@@ -42,9 +42,9 @@ const messages = defineMessages({
     id: 'PatternAnalysis.headerTitle',
     defaultMessage: 'Edit Pattern Rule',
   },
-  editPatternName: {
-    id: 'PatternAnalysis.editPatternName',
-    defaultMessage: 'Edit Pattern Name',
+  createPatternModalDescription: {
+    id: 'PatternAnalysis.createPatternModalDescription',
+    defaultMessage: 'Create a Pattern specifying a condition for common failure reason',
```

Comment:
I see that we already has such a message at `src/pages/inside/projectSettingsPageContainer/content/patternAnalysis/messages.js`, so it can be reused.
Also I'm not sure it's a relevant description for 'Edit modal'. I'll ask the QA team to clarify.

Reply:
I've just discussed it with BA.
Decided to remove this label for "Edit pattern modal" at all.

Reply:
Done.